apply/applydriver/local.go

```diff
@@ -156,7 +156,12 @@ func (c *Applier) reconcileCluster() error {
 	if len(mj) == 0 && len(md) == 0 && len(nj) == 0 && len(nd) == 0 {
 		return c.upgrade()
 	}
-	return c.scaleCluster(mj, md, nj, nd)
+
+	if len(md) > 0 || len(nd) > 0 {
+		logrus.Warnf(`we found these master(%v) or node(%v) are in cluster but not in you Kubefile, we will do nothing for them.`, md, nd)
```

Comment:
If this is a dangerous action, I suggest that we block the execution and let user to realize the gravity of this situation. And let end user to judge whether to add them in Clusterfile or anything else. 

When we do not know what is the better action, we should do it strictly. Otherwise I think it could be all in mess. @VinceCui  WDYT